payment-manager/src/main/java/com/qpay/paymentmanager/service/impl/WalletServiceImpl.java

```diff
@@ -21,18 +22,14 @@ public WalletEntity getWalletById(long id) {
         return walletRepository.findById(id).orElseThrow(NoWalletFoundException::new);
     }
 
-    public WalletEntity addWallet(WalletModification walletModification) {
-        final var walletEntity = walletMapper.map(walletModification);
-
+    public WalletEntity addWallet(WalletCreation walletCreation) {
```

Comment:
Did you set up `final` in your Intellij ? There are lots of places where you miss it